jiratools/__init__.py

```diff
@@ -157,20 +173,26 @@ def cli_jira_link():
         print('ERROR: "{}" trying to make the link.'.format(e.text))
 
 
-def _list_from_config(key_name):
+def _list_from_config(key_name: str) -> list:
```

Comment:
You can do `List[str]`, yeah? I'm generally in favor of being as specific as possible.